### ISSUE TITLE
Pull zipkin address from configmap

### ISSUE THREAD

**Issue by ZackButcher** [istio/mixer#1032]
Envoy gets the address to send zipkin traces from pilot by way of the Istio configmap. Mixer should get its zipkin address in the same way.

**Comment by geeknoid** [comment]
I don't think we want Mixer to have a direct dependency on the config map. Can this be handled through the normal config flow instead?

**Comment by ZackButcher** [comment]
It'll require rewriting a bit of code - we wire up the tracer in cmd/server/server.go right now, but it is possible.
